src/ErrorBox/ErrorBox.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { ExclamationCircleIcon } from "@heroicons/react/solid";
+import { DismissButton } from "../DismissButton/DismissButton";
```

Comment:
Instead of duplicating the component in this repo, import it from decanter-react.